buildenv/jenkins/jobs/infrastructure/artifactoryCheck.groovy

```diff
@@ -0,0 +1,73 @@
+ *******************************************************************************/
+def LABEL = (params.LABEL) ?: "artifactory"
+def WARNING_THRESHOLD = (params.WARNING_THRESHOLD) ? params.WARNING.toInteger() : 75
+def DANGER_THRESHOLD = (params.DANGER_THRESHOLD) ? params.DANGER_THRESHOLD.toInteger() : 90
+def SLACK_CHANNEL = (params.SLACK_CHANNEL) ?: "#jenkins-sandbox"
+def SLACK_MESSAGE = "${JOB_NAME} jenkins job\n"
+def HIGH_ROOT_DISK_PERCENTAGE = 0
+def HIGH_ARTIFACTORY_DISK_PERCENTAGE = 0
+def HIGH_NODE_ROOT
+def HIGH_NODE_ARTIFACTORY
+
+timestamps {
+	stage("Artifactory data collection") {
+		for (sNode in jenkins.model.Jenkins.instance.getLabel(LABEL).getNodes()) {
+			node(sNode.getNodeName()) {
+				// since there are only artifactory nodes in 2 geos, its either one or the other
+				def artifactoryGeo = (sNode.getLabelString().contains("ci.geo.unb")) ? "UNB" : "OSU"
+				def rootDiskDetails = sh (label: "Get / disk details", returnStdout: true, script: "df / -h | tail -1 | awk '{print \$3 \" / \" \$2}'").trim()
+				def rootDiskPercentage = sh (label: "Get / disk percentage", returnStdout: true, script: "df / -h | tail -1 | awk '{print \$5}'").trim()
+				def artifactoryDiskDetails = sh (label: "Get Artifactory disk details", returnStdout: true, script: "df -h | grep jenkins | awk '{print \$3 \" / \" \$2}'").trim()
+				def artifactoryDiskPercentage = sh (label: "Get Artifactory disk percentage", returnStdout: true, script: "df -h | grep jenkins | awk '{print \$5}'").trim()
+				def message = "Artifactory on ${sNode.getNodeName()} hosted at ${artifactoryGeo}:\n  Root disk used: ${rootDiskDetails} -> ${rootDiskPercentage}\n  Data disk used: ${artifactoryDiskDetails} -> ${artifactoryDiskPercentage}\n"
+				SLACK_MESSAGE += "${message}"
+				echo "${message}"
+				if (HIGH_ROOT_DISK_PERCENTAGE < (rootDiskPercentage - '%').toInteger()) {
+					HIGH_ROOT_DISK_PERCENTAGE = (rootDiskPercentage - '%').toInteger()
+					HIGH_NODE_ROOT = sNode.getNodeName()
+				}
+				if (HIGH_ARTIFACTORY_DISK_PERCENTAGE < (artifactoryDiskPercentage - "%").toInteger()) {
+					HIGH_ARTIFACTORY_DISK_PERCENTAGE = (artifactoryDiskPercentage - "%").toInteger()
+					HIGH_NODE_ARTIFACTORY = sNode.getNodeName()
+				}
+			}
+		}
+	}
+
+	stage("Artifactory data review") {
+		def slackStatus = "Good"
+		
+		if ((HIGH_ROOT_DISK_PERCENTAGE > DANGER_THRESHOLD) || (HIGH_ROOT_DISK_PERCENTAGE > DANGER_THRESHOLD)) {
+			slackStatus = "Danger"
+		}
+		if ((HIGH_ARTIFACTORY_DISK_PERCENTAGE > WARNING_THRESHOLD) || (HIGH_ARTIFACTORY_DISK_PERCENTAGE > WARNING_THRESHOLD)) {
+			slackStatus = "Warning"
+		}
+		message = "Highest root disk percentage ${HIGH_ROOT_DISK_PERCENTAGE}% on ${HIGH_NODE_ROOT}\nHighest artifactory disk percentage ${HIGH_ARTIFACTORY_DISK_PERCENTAGE}% on ${HIGH_NODE_ARTIFACTORY}\n"
+		echo "${message}Slack status: ${slackStatus}"
```

Comment:
Does having `\n` at the end of `message` affect the Slack message, vs moving it into the echo (ie. `${message}\nSlack status:`?

Reply:
No impact, easier to read in the second place though.  Modified in the latest update.

https://ci.eclipse.org/openj9/job/joe/224/console
https://openj9.slack.com/archives/CBRG8KTGV/p1611960637006200

Note: `joe jenkins job` turns into `Check_Artifactory_Disk_Usage jenkins job` when it the offical job.